### ISSUE TITLE
Store wallet and user credentials on localStorage

### ISSUE THREAD

**Issue by fiatjaf** [lnbits/lnbits#117]
I thought it would be enough to rely on browser history to access my LNbits wallets, but after creating a hundred wallets and deleting the database some times and creating wallets again I'm pretty sure I will never find the wallet I want ever again.

Since we already leak previously used wallets into the browser history, what if we also stored them on localStorage and presented the user with a list of previously used wallets at the homepage of LNbits?

If someone wants a temporary instawallet they really need to use private-browsing mode or something.

Or maybe this feature could be turned off by the server owner.

**Comment by HamishMacEwan** [comment]
This would give access to wallets created at the browser by the user who created them only?  That may be implicit, but just wanted to be sure.

That being the case, and with my own management issues (greatly helped by the appearance of the `Wallet name` in the <title>) has improved ease of management a great deal.  I wonder then if adding the `Wallet name` to the URL might be another affordance worth considering.

TL;DR Ack lamation.

**Comment by fiatjaf** [comment]
It would give access to the wallets to anyone using that same browser.

**Comment by HamishMacEwan** [comment]
So that's the same risk as auto-login on a shared browser, which may be reasonably well understood.  Unless you're on a public or kiosk browser, not something that's common or impossible to avoid.  Is the localStorage protected by profile login, as in Firefox and others?

As you note there's private mode too.

**Comment by jcspocket** [comment]
It would not be adding any additional risk really.  In some ways making it more transparent that the risk already exists. 

As fiatjaf mentioned, you can already access browser history to get into old wallets.  Users might not be aware of that.

Making it show the previously accessed wallets on the front page makes it more obvious.

**Comment by talvasconcelos** [comment]
Can this be closed? @fiatjaf ?
